dlc-wallet-test/src/test/scala/org/bitcoins/dlc/wallet/DLCMultiOracleNumericExecutionTest.scala

```diff
@@ -185,7 +146,31 @@ class DLCMultiOracleNumericExecutionTest
     }
   }
 
-  it must "execute as the recipient" in { wallets =>
+  /** Builds oracle attestments for the given oracle outcome */
+  private def buildAttestments(
+      oracleOutcomes: NumericOracleOutcome): Vector[OracleAttestmentV0TLV] = {
+    privateKeys.zip(kValues).flatMap { case (priv, kValues) =>
+      val outcomeOpt = oracleOutcomes.oraclesAndOutcomes.find(
+        _._1.publicKey == priv.schnorrPublicKey)
+
+      outcomeOpt.map { case (oracleInfo, outcome) =>
+        val sigs = outcome.digits.zip(kValues).map { case (num, kValue) =>
```

Comment:
if you want to have the invariants then you'll need to extend `outcome.digits` with right padding (say, `0`s, since it doesn't matter what you put there) up to the number of digits specified in the contract info (probably a parameter this `def` will need